test/cri-containerd/main.go

```diff
@@ -26,20 +26,25 @@ import (
 )
 
 const (
-	daemonAddress                     = "tcp://127.0.0.1:2376"
-	connectTimeout                    = time.Second * 10
-	testNamespace                     = "cri-containerd-test"
+	daemonAddress  = "tcp://127.0.0.1:2376"
+	connectTimeout = time.Second * 10
+	testNamespace  = "cri-containerd-test"
+
 	wcowProcessRuntimeHandler         = "runhcs-wcow-process"
 	wcowHypervisorRuntimeHandler      = "runhcs-wcow-hypervisor"
 	wcowHypervisor17763RuntimeHandler = "runhcs-wcow-hypervisor-17763"
 	wcowHypervisor18362RuntimeHandler = "runhcs-wcow-hypervisor-18362"
-	lcowRuntimeHandler                = "runhcs-lcow"
-	imageLcowK8sPause                 = "k8s.gcr.io/pause:3.1"
-	imageLcowAlpine                   = "docker.io/library/alpine:latest"
-	imageLcowCosmos                   = "cosmosarno/spark-master:2.4.1_2019-04-18_8e864ce"
-	testGPUBootFiles                  = "C:\\ContainerPlat\\LinuxBootFiles\\nvidiagpu"
-	alpineAspNet                      = "mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11"
-	alpineAspnetUpgrade               = "mcr.microsoft.com/dotnet/core/aspnet:3.1.2-alpine3.11"
+	testDeviceUtilFilePath            = "C:\\ContainerPlat\\device-util.exe"
```

Comment:
Is this going to be the actual path where device-util lives or not decided? If so I don't think it needs a test prefix

Reply:
Yes this is the path expected. The install path (ie "C:\\ContainerPlat") could be different so this constant is really more to have that value in a central place that's easier to find and update on breaks.

Reply:
Makes sense. I'm going to stop commenting on things past 8 PM 😌